pkg/flowaggregator/flowaggregator.go

```diff
@@ -194,7 +275,28 @@ func (fa *flowAggregator) Run(stopCh <-chan struct{}) {
 					continue
 				}
 			}
-			err := fa.aggregationProcess.ForAllRecordsDo(fa.sendFlowKeyRecords)
+			dataSet := ipfix.NewSet(ipfixentities.Data, fa.templateID, false)
+			addFlowKeyRecord := func(key ipfixintermediate.FlowKey, record ipfixintermediate.AggregationFlowRecord) error {
+				if !record.ReadyToSend {
+					klog.V(4).Info("Skip sending record that is not correlated.")
+					return nil
+				}
+				// limit dataSet to have no more than two records
+				// TODO: more records per dataSet will be supported when go-ipfix supports size check when adding records
+				if dataSet.GetSet().GetNumberOfRecords() == recordNumPerSet {
+					_, err := fa.sendDataSet(dataSet)
+					if err != nil {
+						return err
+					}
+					dataSet = ipfix.NewSet(ipfixentities.Data, fa.templateID, false)
```

Comment:
We still need to do the following deletion right? I think we should do only one record per set and delete as soon as we send it. We can still use `sendDataSet` and delete `sendFlowKeyRecords`.

```
// TODO: We will delete only inactive flows when intermediate process supports it in go-ipfix.
	fa.aggregationProcess.DeleteFlowKeyFromMapWithoutLock(key)		fa.aggregationProcess.DeleteFlowKeyFromMapWithoutLock(key)
```